ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/Database.scala

```diff
@@ -211,7 +211,10 @@ object Database {
         .placeholders(Map("table.prefix" -> TablePrefix).asJava)
         .table(TablePrefix + Flyway.configure().getTable)
         .dataSource(adminConnectionPool)
-        .locations(s"classpath:/com/daml/ledger/on/sql/migrations/${system.name}")
+        .locations(
+          "classpath:/com/daml/ledger/on/sql/migrations/common",
```

Comment:
Sharing migrations across sql dialects means we now share the maximum migration number as well, right?

Reply:
Yup, absolutely. I think this is probably a good thing.